README.md

```diff
@@ -1,14 +1,60 @@
 # Data Analysis
-This repository contains data and code associated with analysis of data gathered from the network of sensor boxes in East Boston.
+Various tools for QuantAQ sensor data analysis. See `notebooks/` for the older version of this repo which contains jupyter notebooks for visual data analysis.
 
-## Contents
-Current contents of this repository include two notebooks associated with some exploration over Spring 2020 ([looking at the raw data](https://github.com/airpartners/data-analysis/blob/master/SP19-raw-data-analysis-test.ipynb) and [doing some pandas processing](https://github.com/airpartners/data-analysis/blob/master/SP19-data-analysis-test.ipynb)) and some [preliminary AQI computations and analysis](https://github.com/airpartners/data-analysis/blob/master/aqi-analysis.ipynb) [associated with an assumpion test](https://docs.google.com/document/d/1EtGzySFGZ5PVPhCsywW77AdkwYG_axO-Zp_cfnLsTVg/edit?usp=sharing) conducted in Fall 2020.
+## Pipeline
+This pipeline automates the data pulling and cleaning process for grabbing data from the QuantAQ network of sensors.
+### Prerequisites
+1. These instructions assume you are running at least Python 3.8
+2. Run `pip install -r requirements.txt` to install the Python dependencies.
```

Comment:
Cool, since I'd like this pipeline to allow a bit of separation between pulling/cleaning data and visualizing it, I pulled out the `rpy2` dependency into its own `pip install` step and specified that R needs to be installed in order to make any visualizations. I did end up preserving the order, just to keep the "optional" steps last. without including the `rpy2` dependency in `requirements.txt`, it should be possible to at least complete the Python installation without installing R first. I'll do a small refactor as well to make sure that the pipeline script can be run without the R dependency